README.md

```diff
@@ -965,8 +912,8 @@ class Container {
 
     const { public, private } = instance[Symbol.metadata][INJECTIONS];
 
-    for (let key in public) {
-      instance[key] = this.lookup(public[key].injectionKey);
+    for (let { injectionKey, set } in public) {
+      set.call(instance, this.lookup(injectionKey))
```

Comment:
for data properties, presumably `set` would do a [[Set]], iow, be `function (v) { this[prop] = v; }`?

Reply:
Yes, correct

Reply:
built-in lenses, i like it, picasso